src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs

```diff
@@ -7,12 +7,17 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using ReactiveUI;
 
 namespace GitHub.ViewModels
 {
     [ExportViewModel(ViewType = UIViewType.PRCreation)]
     [PartCreationPolicy(CreationPolicy.NonShared)]
     class PullRequestCreationViewModel : BaseViewModel, IPullRequestCreationViewModel
     {
+        public PullRequestCreationViewModel()
+        {
+            CancelCommand = ReactiveCommand.Create();
```

Comment:
Let's move this to the `BaseViewModel` ctor and dedupe some code along the way (and avoid this problem in the future)